src/app/wallets/get-on-chain-fee.ts

```diff
@@ -139,6 +139,13 @@ const getOnChainFee = async <S extends WalletCurrency, R extends WalletCurrency>
     "payOnChainByWalletId.estimatedMinerFee": `${paymentFlow.btcMinerFee}`,
   })
 
+  // avoids lnd balance sniffing attack
+  const balanceCheck = paymentFlow.checkBalanceForSend({
+    amount: balance.amount * 2n,
```

Comment:
If balance is low, thing 10k sats, this check, therefore getFee might error. But sendAll will actually work. Add some constant value, ie 30k sats, will remove the issue